static/css/post.less

```diff
@@ -413,6 +413,36 @@
 .post_body {
     .u-clearfix();
     color: @darkgray;
+
+    // Spacing has been eyeballed to be 40px.
+    p ~ h1, p ~ .h1 {
+        margin-top: unit(22px / @base-font-size-px, em);
+    }
+
+    // Spacing has been eyeballed to be 40px.
+    p ~ h2, p ~ .h2 {
+        margin-top: unit(31px / @base-font-size-px, em)
+    }
+
+    // Spacing has been eyeballed to be 40px.
+    p ~ h3, p ~ .h3 {
+        margin-top: unit(35px / @base-font-size-px, em)
+    }
+
+    // Spacing has been eyeballed to be 15px.
+    p ~ h4, p ~ .h4 {
+        margin-bottom: unit(7px / @base-font-size-px, em);
```

Comment:
Doesn't have to happen here, but I think eventually we'll want to work out a pattern for doing optical adjustments.
It would be nice to have in the code the actually intended gap width value, with the optical offset added separately. So you could easily find in a search which blocks were intended to have the same gap width.

Pseudocode of the general idea:
```
@optical-adjustment: 8px;
margin-top: unit( (15px - @optical-adjustment) / @base-font-size-px, em)
```

(That's just the general idea, the difficulty is the optical-adjustments appear to be a bunch of different values)